src/plugin/kubearmor.go

```diff
@@ -50,6 +50,61 @@ func ConvertKnoxSystemPolicyToKubeArmorPolicy(knoxPolicies []types.KnoxSystemPol
 	return results
 }
 
+func ConvertSQLiteKubeArmorLogsToKnoxSystemLogs(docs []map[string]interface{}) []types.KnoxSystemLog {
```

Comment:
This function is not used as well from anywhere.

Reply:
This function is used here https://github.com/ErenYeager713/auto-policy-discovery/blob/dev/src/systempolicy/systemPolicy.go